content/docs/dvclive/live/log_artifact.md

```diff
@@ -26,6 +39,15 @@ Uses `dvc add` to track `path` with DVC, generating a `{path}.dvc` file.
 When combined with `save_dvc_exp=True`, it will ensure that `{path}.dvc` is
 included in the experiment.
 
+If `Live` was initialized with `dvcyaml=True` (which is the default), it will
+add an [artifact](/doc/user-guide/project-structure/dvcyaml-files#artifacts) and
+all the metadata passed as arguments to corresponding `dvc.yaml`. Passing
+`type="model"` will mark it as a `model` for DVC and will make it appear in
```

Comment:
... for DVC and will also make [Studio Model Registry](/doc/studio) support it (coming soon!).

Same as above